misc-scripts/xref_mapping/XrefParser/HGNCParser.pm

```diff
@@ -15,8 +15,55 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=head1 NAME
+
+Bio::EnsEMBL::Xref::Parser::HGNCParser
```

Comment:
This and other occurrences of _Bio::EnsEMBL::Xref::Parser_ in POD should be changed to _XrefParser_